tests/integrationtests/jvm/JvmBasicFunctionalTests.kt

```diff
@@ -41,6 +41,27 @@ class JvmBasicFunctionalTests : AssertionTestCase("tests/integrationtests/jvm/ba
         }
     }
 
+    @Test
+    fun testJarNormalization() {
+        jarTestCase(
+            name = "test_module_name_lib.jar",
+            description = "Builder jars should be normalized with the same timestamps as singlejar and including stamp data"
+        ) {
+            validateFileSha256("513d14b29eb1b95b97bf7d34e2126a716c7d1012b259b5021c16b99ca82feeb5")
```

Comment:
This should be a valid way of validating the jar right ? across operating systems ?